lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -140,6 +143,20 @@ def hub_release_exists(semver)
       false
     end
 
+    def hub_find_remote_tag(tag_name)
+      attempts = 0
+      # Attempt to find the build. Rescue and re-attempt if the build can not
+      # be found on github yet.
+      begin
+        sleep 1 # Give some time for tag to showup.
```

Comment:
Should we move this into the rescue so that we only wait for the tag if it isn't there on the first attempt?